### ISSUE TITLE
A false warning thrown by oscapd-evaluate in atomic scan

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap-daemon#103]
When we run `atomic scan`, we don't have docker installed inside the `openscap` container 😬 
Therefore the warning that container scanning is disabled is a false warning, because we actually scan a container, just oscapd doesn't know about it.

```
[root@thinkpad atomic]# atomic  scan --verbose a2d9f633eaab
docker run -t --rm -v /etc/localtime:/etc/localtime -v /run/atomic/2017-06-06-14-08-22-857365:/scanin -v /var/lib/atomic/openscap/2017-06-06-14-08-22-857365:/scanout:rw,Z -v /etc/oscapd:/etc/oscapd:ro openscap-docker-testing oscapd-evaluate scan --no-standard-compliance --targets chroots-in-dir:///scanin --output /scanout -j1
INFO:OpenSCAP Daemon one-off evaluator 0.1.7
WARNING:Can't import the 'docker' package. Container scanning functionality will be disabled.
...
...
```

**Comment by mpreisler** [comment]
The warning is confusing but it is technically correct.

**Comment by mpreisler** [comment]
Fixed by 05535ac8f0a07acbabe9147546999bd054050c90
